app/bundles/CoreBundle/Controller/MauticController.php

```diff
@@ -2,21 +2,9 @@
 
 namespace Mautic\CoreBundle\Controller;
 
-use Mautic\CoreBundle\Factory\ModelFactory;
-use Symfony\Component\HttpKernel\Event\ControllerEvent;
-
 /**
  * A dummy interface to ensure that only Mautic bundles are affected by Mautic onKernelController events.
  */
 interface MauticController
```

Comment:
I think we can remove this interface completelly as it's not used bu removeing the `onKernelController` method

Reply:
I plan on using it later for autowiring purposes. But sure can remove it and add later. WDYT?

Reply:
If you plan to use it then it's OK to leave it there.